commands/inventory/add_agent_rds_exporter.go

```diff
@@ -24,14 +24,16 @@ import (
 
 var addAgentRDSExporterResultT = commands.ParseTemplate(`
 RDS Exporter added.
-Agent ID              : {{ .Agent.AgentID }}
-PMM-Agent ID          : {{ .Agent.PMMAgentID }}
-Service ID            : {{ .Agent.ServiceID }}
-Listen port           : {{ .Agent.ListenPort }}
+Agent ID                  : {{ .Agent.AgentID }}
+PMM-Agent ID              : {{ .Agent.PMMAgentID }}
+Node ID                   : {{ .Agent.NodeID }}
```

Comment:
According to https://github.com/percona/pmm/pull/500/files#diff-d8f7a0a7d4635c8a9d269712db67f417R304